bubble-wrap.yaml

```diff
@@ -959,7 +958,7 @@ styles:
 
 scene:
     background:
-        color: global.scene1
+        color: [0.870,0.870,0.870]
```

Comment:
Fixed in https://github.com/tangrams/bubble-wrap/pull/204/commits/8420e66c6820d9030495c9202a96340e2f06ce54.